wire-ios-data-model/Tests/MLS/CoreCryptoConfigProviderTests.swift

```diff
@@ -55,7 +55,31 @@ class CoreCryptoFactoryTests: ZMConversationTestsBase {
 
     // MARK: - Core crypto configuration
 
-    func test_itReturnsCoreCryptoConfiguration() throws {
+    func test_itReturnsInitialCoreCryptoConfiguratio() throws {
+        try syncMOC.performGroupedAndWait { context in
+            // GIVEN
+            let selfUser = ZMUser.selfUser(in: context)
+            selfUser.remoteIdentifier = UUID.create()
+
+            // mock core crypto key
+            let key = Data([1, 2, 3])
+            self.mockCoreCryptoKeyProvider.coreCryptoKeyMock = {
+                return key
+            }
+
+            // WHEN
+            let configuration = try self.sut.createInitialConfiguration(
+                sharedContainerURL: OtrBaseTest.sharedContainerURL,
+                selfUser: selfUser
+            )
+
+            // THEN
+            XCTAssertEqual(configuration.key, key.base64EncodedString())
+            XCTAssertEqual(configuration.path, self.expectedPath(selfUser))
+        }
+    }
+
+    func test_itReturnsFullCoreCryptoConfiguration() throws {
```

Comment:
method name does not match the test

Reply:
test_createFullConfigurationThrowsErrorsIfAlreadyCreated ?

Reply:
the test name is correct, it's just that github auto-collapsed two test methods together